### DISCUSSION TITLE
Change terminal default to Powershell?

### DISCUSSION THREAD

**Discussion by ghost** [Ylianst/MeshCentral#4989] [General]
Is it possible to change the default terminal prompt from Admin Shell to Admin Powershell?

**Comment by silversword411** [comment]
No

**Comment by The-Real-Admin-Guy** [comment]
You can actually invoke powershell from within the console in Windows 10/11. It works just fine.   
Windows 11 is a monumental pain, for remote admins via desktop and using powershell.   
I use the console but switch it to powershell all the time, and it works just fine.

 Just remember to exit powershell and return to regular console  prior to terminating the session or it could cause issues where you may need to reinstall the agent...

**Comment by si458** [comment]
right click the connect button and select `powershell`
so i dont believe the is a way at the moment to change the default terminal at the moment

**Comment by amoljagdalepucsd** [comment]
Hi @RichardBurt65 @si458 
I think we can do it by doing some custom changes.
Inside connect Termina function we can add Powershell's protocol (6) value to options object which we are getting function parameter to make it  Powershell by default.
function connectTerminal(e, contype, options) { 
       options = {
                        protocol : 6, 
                        ... exising keys and values
                    }
}

can you please try it
